otx/core/data/action_dataset_adapter.py

```diff
@@ -0,0 +1,199 @@
+        """Convert DatumaroDataset to DatasetEntity for Acion Classification."""
+        label_information = self._prepare_label_information(datumaro_dataset)
+        label_entities = label_information["label_entities"]
+
+        label_schema = self._generate_default_label_schema(label_entities)
+        dataset_items = []
+        for subset, subset_data in datumaro_dataset.items():
+            for datumaro_items in subset_data:
+                for datumaro_item in datumaro_items:
+                    image = Image(file_path=datumaro_item.media.path)
+                    shapes = []
+                    for ann in datumaro_item.annotations:
+                        # Action Classification
+                        if isinstance(ann, DatumaroLabel):
+                            shapes.append(
+                                Annotation(
+                                    Rectangle.generate_full_box(), labels=[ScoredLabel(label=label_entities[ann.label])]
+                                )
+                            )
+                    meta_item = MetadataItemEntity(
+                        data=VideoMetadata(
+                            name="video_meta",
+                            video_id=int(datumaro_item.media.path.split("/")[-3].split("_")[-1]),
+                            frame_idx=int(datumaro_item.media.path.split("/")[-1].split(".")[0].lstrip("0")),
+                        )
+                    )
+                    # Unlabeled dataset
+                    annotation_scene = None  # type: Any
+                    if len(shapes) == 0:
+                        annotation_scene = NullAnnotationSceneEntity()
+                    else:
+                        annotation_scene = AnnotationSceneEntity(
+                            kind=AnnotationSceneKind.ANNOTATION, annotations=shapes
+                        )
+                    dataset_item = DatasetItemEntity(image, annotation_scene, subset=subset, metadata=[meta_item])
+                    dataset_items.append(dataset_item)
+        return DatasetEntity(items=dataset_items), label_schema
+
+
+class ActionDetectionDatasetAdapter(ActionBaseDatasetAdapter):
+    """Action Detection adapter inherited by ActionBaseDatasetAdapter and BaseDatasetAdapter."""
+
+    def convert_to_otx_format(self, datumaro_dataset: dict) -> Tuple[DatasetEntity, LabelSchemaEntity]:
+        """Convert DatumaroDataset to DatasetEntity for Acion Detection."""
+        label_information = self._prepare_label_information(datumaro_dataset)
+        label_entities = label_information["label_entities"]
+
+        label_schema = self._generate_default_label_schema(label_entities)
+        dataset_items = []
+        for subset, subset_data in datumaro_dataset.items():
+            for datumaro_items in subset_data:
+                for datumaro_item in datumaro_items:
+                    image = Image(file_path=datumaro_item.media.path)
```

Comment:
It would be nice if this part (for statement) was also subtracted as a function.